app/controllers/application_controller.rb

```diff
@@ -14,6 +15,16 @@ def switch_language
 
   private
 
+  def error_server
+    flash[:danger] = t "server.error_message"
+    redirect_to root_path
+  end
+
+  def find_by_id_not_found
+    flash[:danger] = t "not_found.record"
+    redirect_to root_path
+  end
```

Comment:
Chưa thấy hợp lý lắm nha. Lỗi Record not found chung quá. Trong khi người dùng không biết record là gì